scripts/linters.sh

```diff
@@ -1,3 +1,3 @@
 mdspell --en-us --ignore-acronyms --ignore-numbers --no-suggestions --report *.md */*.md */*/*.md */*/*/*.md */*/*/*/*.md
```

Comment:
there should be a
```
#! /bin/sh
# This runs 3 checks.... [some doc/header]
set -x
set -e
```
the set -e is to stop on first error, set -x to see what it is doing

Reply:
edit: but it's good indeed to separate the stages in circle so it's more clear what it is doing and where it may fail/stop